### PR TITLE
[mod_whosonline]XML code style

### PR DESCRIPTION
**Pull request by wojsmol**
### Summary of Changes

Code style changes for mod_whosonline
### Testing Instructions

code review

cc @JoomliC @andrepereiradasilva

### CODE REVIEW

File: modules/mod_whosonline/mod_whosonline.xml [hunk 1]
```diff
@@ -25,45 +25,49 @@
...
+
 				<field
 					name="filter_groups"
 					type="radio"
 					class="btn-group btn-group-yesno"
 					default="0"
 					label="MOD_WHOSONLINE_FIELD_FILTER_GROUPS_LABEL"
-					description="MOD_WHOSONLINE_FIELD_FILTER_GROUPS_DESC">
+					description="MOD_WHOSONLINE_FIELD_FILTER_GROUPS_DESC"
+					>
```
Comment (andrepereiradasilva) [inline modules/mod_whosonline/mod_whosonline.xml]: shouldn't label and description be moved up?

File: modules/mod_whosonline/mod_whosonline.xml [hunk 2]
```diff
@@ -25,45 +25,49 @@
...
 					name="cache"
 					type="list"
 					default="0"
 					label="COM_MODULES_FIELD_CACHING_LABEL"
-					description="COM_MODULES_FIELD_CACHING_DESC">
-					<option
-						value="0">COM_MODULES_FIELD_VALUE_NOCACHING</option>
+					description="COM_MODULES_FIELD_CACHING_DESC"
+					>
+					<option value="0">COM_MODULES_FIELD_VALUE_NOCACHING</option>
```
Comment (andrepereiradasilva) [inline modules/mod_whosonline/mod_whosonline.xml]: shouldn't label and description be moved up?

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on 2156437efbf09b21ca3167cc5a24bf04c59dbc8c<br /><br />on code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10597">issues.joomla.org/joomla-cms/10597</a>.</sub>

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 2156437efbf09b21ca3167cc5a24bf04c59dbc8c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10597">issues.joomla.org/joomla-cms/10597</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10597">issues.joomla.org/joomla-cms/10597</a>.</sub>

**Comment by brianteeman** [comment]
oops chose the wrong option<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10597">issues.joomla.org/joomla-cms/10597</a>.</sub>

**Pull request closed without merging**